scripts/pyxtal/pyxtal_vs_pymatgen.py

```diff
@@ -0,0 +1,23 @@
+from argparse import ArgumentParser
+
+from pymatgen.symmetry.groups import (
+    PointGroup,
+    SpaceGroup,
+    SymmetryGroup,
+    sg_symbol_from_int_number,
+)
+from pyxtal.symmetry import Group
+
+N_SYMMETRY_GROUPS = 230
+
+for idx in range(1, N_SYMMETRY_GROUPS + 1):
+    sg_int = sg_symbol_from_int_number(idx)
+    sg_pymatgen = SpaceGroup(sg_int)
+    sg_pyxtal = Group(idx)
+    if sg_pymatgen.symbol != sg_pyxtal.symbol or sg_pyxtal.number != idx:
+        print(f"idx: {idx}")
+        print(f"sg_int: {sg_int}")
+        print(f"sg_pyxtal_number: {sg_pyxtal.number}")
+        print(f"pymatgen symbol: {sg_pymatgen.symbol}")
+        print(f"pyxtal symbol: {sg_pyxtal.symbol}")
+        print("---")
```

Comment:
I am not sure what this script aims to do. Find differences between the space groups defined in pymatgen and pyxtal? It would be nice to have at least a script-level comment to say what the goal of this script is.

Reply:
This is my initial exploration script of PyXtal. I first wanted to make sure that the identification of space groups was compatible with pymatgen. I thought it could be ok to put it in the repo. I will add a little description.

Reply:
Done